src/pretalx/submission/models/submission.py

```diff
@@ -12,6 +12,10 @@
 from pretalx.mail.context import template_context_from_submission
 
 
+def generate_invite_code(length=32):
+    return get_random_string(length=length, allowed_chars=Submission.CODE_CHARSET)
```

Comment:
Note that the orga team invite also generates tokens for invite links. There are now at least three places in pretalx, which do that. Maybe think about centralizing this in the future.

Reply:
Orga invites work entirely differently. Orga invites pre-create a user with the correct permissions. But since we don't have a permission set for speakers as such, I just used a shared invite token.